src/AcmTable/AcmTable.tsx

```diff
@@ -360,7 +399,26 @@ export function AcmTable<T>(props: AcmTableProps<T>) {
     }>(() => {
         /* istanbul ignore if */
         if (!items) return { tableItems: [], totalCount: 0 }
-        const tableItems = items.map((item) => {
+        let filteredItems: T[] = items
+        if (Object.keys(toolbarFilters).length > 0) {
+            const filterCategories = Object.keys(toolbarFilters)
+            filteredItems = items.filter((item: T) => {
+                let isFilterMatch = true
+                // Item must match 1 filter of each category
+                filterCategories.forEach((filter: string) => {
+                    const filterItem: ITableFilterItem<T> | undefined = filterItems.find(
+                        (filterItem) => filterItem.id === filter
+                    )
+                    /* istanbul ignore next */
+                    const isMatch = filterItem?.tableFilterFn(get(toolbarFilters, `${filter}`), item) ?? true
+                    if (!isMatch) {
+                        isFilterMatch = false
```

Comment:
if there is a match, we should short circut for performance.  cant do that in a .forEach, so change that.